src/printer.js

```diff
@@ -1053,10 +1053,11 @@ function printExpression(path, options, print) {
             : ""
         ]);
       case "inline":
-        return join(
+        /*return join(
           literalline,
           node.raw.replace("___PSEUDO_INLINE_PLACEHOLDER___", "").split("\n")
-        );
+        );*/
+        return node.raw;
```

Comment:
Use `return join(literalline, node.raw.split("\n"))` always, otherwise you break indent